### DISCUSSION TITLE
Last versions can't work create_all

### DISCUSSION THREAD

**Discussion by richard24se** [sqlalchemy/sqlalchemy#7361] [Usage Questions]
Hello I have this error:

sqlite3.OperationalError: no such table: pdlc_laplr

I was call create_all but doesn't create tables

**Comment by CaselIT** [comment]
Hi,

without an example that reproduces the issue it's impossible to offer help. Look for issues with "great mcve" for examples on how to report an issue. https://github.com/sqlalchemy/sqlalchemy/issues?q=is%3Aissue+label%3A%22great+mcve%22+
